pages/links.md

```diff
@@ -12,13 +12,10 @@ permalink: /pages/resources/links/
       <h1>Resources</h1>
       <p><a href="http://www.firstinspires.org" target="_blank">FIRST</a> &#8211; For Inspiration and Recognition of Science and Technology &#8211;The parent organization.</p>
       <p><a href="https://www.firstinspires.org/robotics/frc">FRC</a> &#8211; FIRST Robotics Competition, the competition that we compete in.</p>
-      <p><a href="https://drive.google.com/drive/folders/0B7KsM0ePNYpJbTd3VjFpTEhLYVU?usp=sharing"
```

Comment:
Is the Drive link deleted? I don't see it moved. Why are we deleting the Drive link?

Reply:
Hi @L5eoneill, there are two drives that I found:

1. Old one that is currently linked: https://drive.google.com/drive/folders/0B7KsM0ePNYpJbTd3VjFpTEhLYVU
2. New one that we've been using: https://drive.google.com/drive/u/0/folders/0AHn98DdEIogOUk9PVA

I removed the drive link because it's an internal resource, so I didn't think needed to be publicized to all.  

If you find it useful, we should link to the new one instead.  

As a separate effort, we should probably combine the two.